x-pack/legacy/plugins/monitoring/server/lib/setup/collection/get_collection_status.js

```diff
@@ -285,6 +311,25 @@ export const getCollectionStatus = async (req, indexPatterns, clusterUuid) => {
     const internalCollectorsUuidsMap = {};
     const partiallyMigratedUuidsMap = {};
 
+    if (product.name === ELASTICSEARCH_CUSTOM_ID && liveEsNodes.length) {
+      productStatus.byUuid = liveEsNodes.reduce((accum, esNode) => ({
+        ...accum,
+        [esNode.id]: {
+          node: esNode,
+          isNetNewUser: true
+        },
+      }), {});
+    }
+
+    if (product.name === KIBANA_SYSTEM_ID && liveKibanaInstance) {
+      productStatus.byUuid = {
+        [get(liveKibanaInstance, 'kibana.uuid')]: {
```

Comment:
Are you using a `get()` here incase it does not exist? Wouldn't the key be `undefined` then?

Reply:
It's definitely not guaranteed to be there so I'm just trying to be defensive

Reply:
Right, so if it's not guaranteed to be there you might end up with something like:
```javascript
[undefined]: {...
```
Might be same issue for: https://github.com/elastic/kibana/pull/39832/files/6a9a348d871795b272c683b5cc2bbdc1040591f1#diff-a3c36eaa7da8d988333f17352cfce93cR154

Maybe add a backup key? eg: `[get(liveKibanaInstance, 'kibana.uuid') || 'uuid']: {...`

Or, maybe not even add that property all together if the `uuid` is not there?

Just an observation and some nitpicking. I might be totally wrong here (I often am)